Riot/ViewController/MasterTabBarController.m

```diff
@@ -145,8 +147,8 @@ - (void)viewDidAppear:(BOOL)animated
     else
     {
         // Check whether the user has been already prompted to send crash reports.
-        // (Check whether 'enableCrashReport' flag has been set once)
-        if (![[NSUserDefaults standardUserDefaults] objectForKey:@"enableCrashReport"])
+        // (Check whether 'enableCrashReport' flag has been set once)        
+        if (!RiotSettings.shared.enableCrashReport)
```

Comment:
`enableCrashReport` has indeed 3 states: `Not defined`, `NO` and `YES. We want to check `Not defined` on this line.